packages/openneuro-components/src/facets/FacetRadio.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { AccordionTab } from '../accordion/AccordionTab'
+import { AccordionWrap } from '../accordion/AccordionWrap'
+import { RadioGroup } from '../radio/RadioGroup'
+import './facet.scss'
+
+export interface FacetRadioProps {
+  radioArr: {
+    label: string
+    onChange?: React.MouseEventHandler<HTMLInputElement>
+    checked: boolean
```

Comment:
Rather than using a `checked` key on each item, this input would take a `<facet>_selected` value from the searchParams context, which would be one of the `value`s in `showDatasetsRadio`
```suggestion
```